### ISSUE TITLE
Simultaneous assembly  of two similar genomes

### ISSUE THREAD

**Issue by lamenace91** [ruanjue/wtdbg2#228]
Hello,

The purpose of our project is to assemble the genome of a parasitic unicellular eukaryotic by using ONT reads. 

Some preliminary analyses based on short reads  indicate that our DNA samples and sequence libraries actually include two related genomes/species with similar length (approx 10Mb) and 10-15% of genomic divergence. Moreover, we know that one species is more prevalent than the other one into the samples (approx 2/3 versus 1/3 of DNA).

Our first assemblies with wtdbg2 and default settings produced a genome of 11Mb.This genome is similar to the most frequent genome  into the samples. 

We are wondering if there is a way to get the assembly of both genomes and particularly for the less frequent species?

**Comment by ruanjue** [comment]
wtdbg2 tends to collapse similar genomic sequences. Maybe you can choose CANU, Flye to try. If want to have a test on wtdbg2, please try to increase `-s`  and `-l`, which might be useful in distinguishing similar seuqences.

**Comment by lamenace91** [comment]
Thanks a lot for the quick answer. I will have a try at these parameters. My genomes are small so I should be able to explore different set of values (not so long to run an assembly) to see if I can get a  better result.

**Comment by lamenace91** [comment]
Hello, I wanted to let you know that I ran different assemblies with "-s" varying from 0.05 to 0.5 (steps: 0.05) and "-l" varying from 200 to 400 (steps: 50) and I obtained an assembly with two genomes with the expected size and the expected divergence. A value of "-s" around 0.3 seems to be the key parameter/value in my case since I got similar assemblies with the different values tested for "-l". I still have some work before publication but it's on the good way ... :-)
Thanks again for your help.
